metricbeat/module/windows/perfmon/defs_pdh_windows.go

```diff
@@ -105,92 +105,92 @@ const (
-	PDH_ENTRY_NOT_IN_LOG_FILE:                  struct{}{},
-	PDH_DATA_SOURCE_IS_LOG_FILE:                struct{}{},
-	PDH_DATA_SOURCE_IS_REAL_TIME:               struct{}{},
-	PDH_UNABLE_READ_LOG_HEADER:                 struct{}{},
-	PDH_FILE_NOT_FOUND:                         struct{}{},
-	PDH_FILE_ALREADY_EXISTS:                    struct{}{},
-	PDH_NOT_IMPLEMENTED:                        struct{}{},
-	PDH_STRING_NOT_FOUND:                       struct{}{},
-	PDH_UNABLE_MAP_NAME_FILES:                  struct{}{},
-	PDH_UNKNOWN_LOG_FORMAT:                     struct{}{},
-	PDH_UNKNOWN_LOGSVC_COMMAND:                 struct{}{},
-	PDH_LOGSVC_QUERY_NOT_FOUND:                 struct{}{},
-	PDH_LOGSVC_NOT_OPENED:                      struct{}{},
-	PDH_WBEM_ERROR:                             struct{}{},
-	PDH_ACCESS_DENIED:                          struct{}{},
-	PDH_LOG_FILE_TOO_SMALL:                     struct{}{},
-	PDH_INVALID_DATASOURCE:                     struct{}{},
-	PDH_INVALID_SQLDB:                          struct{}{},
-	PDH_NO_COUNTERS:                            struct{}{},
-	PDH_SQL_ALLOC_FAILED:                       struct{}{},
-	PDH_SQL_ALLOCCON_FAILED:                    struct{}{},
-	PDH_SQL_EXEC_DIRECT_FAILED:                 struct{}{},
-	PDH_SQL_FETCH_FAILED:                       struct{}{},
-	PDH_SQL_ROWCOUNT_FAILED:                    struct{}{},
-	PDH_SQL_MORE_RESULTS_FAILED:                struct{}{},
-	PDH_SQL_CONNECT_FAILED:                     struct{}{},
-	PDH_SQL_BIND_FAILED:                        struct{}{},
-	PDH_CANNOT_CONNECT_WMI_SERVER:              struct{}{},
-	PDH_PLA_COLLECTION_ALREADY_RUNNING:         struct{}{},
-	PDH_PLA_ERROR_SCHEDULE_OVERLAP:             struct{}{},
-	PDH_PLA_COLLECTION_NOT_FOUND:               struct{}{},
-	PDH_PLA_ERROR_SCHEDULE_ELAPSED:             struct{}{},
-	PDH_PLA_ERROR_NOSTART:                      struct{}{},
-	PDH_PLA_ERROR_ALREADY_EXISTS:               struct{}{},
-	PDH_PLA_ERROR_TYPE_MISMATCH:                struct{}{},
-	PDH_PLA_ERROR_FILEPATH:                     struct{}{},
-	PDH_PLA_SERVICE_ERROR:                      struct{}{},
-	PDH_PLA_VALIDATION_ERROR:                   struct{}{},
-	PDH_PLA_VALIDATION_WARNING:                 struct{}{},
-	PDH_PLA_ERROR_NAME_TOO_LONG:                struct{}{},
-	PDH_INVALID_SQL_LOG_FORMAT:                 struct{}{},
-	PDH_COUNTER_ALREADY_IN_QUERY:               struct{}{},
-	PDH_BINARY_LOG_CORRUPT:                     struct{}{},
-	PDH_LOG_SAMPLE_TOO_SMALL:                   struct{}{},
-	PDH_OS_LATER_VERSION:                       struct{}{},
-	PDH_OS_EARLIER_VERSION:                     struct{}{},
-	PDH_INCORRECT_APPEND_TIME:                  struct{}{},
-	PDH_UNMATCHED_APPEND_COUNTER:               struct{}{},
-	PDH_SQL_ALTER_DETAIL_FAILED:                struct{}{},
-	PDH_QUERY_PERF_DATA_TIMEOUT:                struct{}{},
+	PDH_CSTATUS_VALID_DATA:                     {},
```

Comment:
That seems OK. In a future PR, the [template](https://github.com/elastic/beats/blob/master/metricbeat/module/windows/perfmon/mkpdh_defs.go#L54) I wrote can be fixed to not include the `struct{}` part. My `gofmt` version didn't do this change for me, but I ran it inside the Vagrant box which I think has an older Go version.